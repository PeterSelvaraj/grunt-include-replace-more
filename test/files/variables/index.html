{{ var $size-1: "small" }}
{{ var $size-2: "medium" }}
{{ var $size-3: "large" }}

<p>My cat is $size-1.</p>

<p>My dog is $size-2.</p>

<p>My horse is $size-3.</p>

<p>But all of them have things that are $size-1, $size-2 and $size-3 in relation to their own size.</p>

{{ include('inc.html', {"biggest": "$size-3"}) }}