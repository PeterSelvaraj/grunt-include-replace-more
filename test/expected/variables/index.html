



<p>My cat is small.</p>

<p>My dog is medium.</p>

<p>My horse is large.</p>

<p>But all of them have things that are small, medium and large in relation to their own size.</p>

<hr />

<p>But which is the biggest size? large</p>